ui/src/message_popup/author/new_question_page.jsx

```diff
@@ -0,0 +1,14 @@
+/* @flow weak */
+import React from 'react';
+
+import QuestionPage from './question_page.jsx';
```

Comment:
This component looks like it doesn't do anything, would it make sense to just call `QuestionPage` directly?